lib/elements/elements.dart

```diff
@@ -560,7 +560,7 @@ class TabController {
   }
 
   TabElement get selectedTab =>
-      tabs.firstWhere((tab) => tab.hasAttr('selected'));
+      tabs.firstWhere((tab) => tab.hasAttr('selected'), orElse: () => null);
 
```

Comment:
Can you help me understand this change? Were you seeing an exception thrown here? 

There *should* always be a selected tab. I'm worried this change could be masking over a timing issue.

Reply:
Earlier the tabs only has `aria-selected` attribute and not `selected`. So it was throwing `Bad state` error for the first time load. I realised after adding `selected` attribute to the default tab, this null check is not required anymore. 😉 Updated PR.